src/renderers/testing/__tests__/ReactTestRenderer-test.js

```diff
@@ -231,6 +232,84 @@ describe('ReactTestRenderer', () => {
     );
   });
 
+  it('allows an optional getMockRef function', () => {
```

Comment:
Let’s also test that if you pass `{}` as options, `getMockRef` will be returning `null` (as opposed to throwing on `options.getMockRef()`).